src/playground/dto/private-bid-params.dto.ts

```diff
@@ -0,0 +1,13 @@
+import { IsString, IsNumberString } from 'class-validator';
+import { IsOptionalString } from 'src/decorators/IsOptionalString';
+
+export class PrivateBidParams {
+  @IsString()
+  mintAccount: string;
```

Comment:
We have a `IsSolanaAddress()` decorator which check if the string is an address on curve